source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Test Retries
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+
+- What are test retries?
+- Why is test retries important?
```

Comment:
It seems weird to have "What *are* test retries" and "Why *is* test retries important" together. Is "test retries" the feature plural or singular?

Reply:
Good catch @CypressCecelia!